dts/bindings/test/vnd,i3c.yaml

```diff
@@ -9,3 +9,4 @@ compatible: "vnd,i3c"
 
 include:
   - name: i3c-controller.yaml
+  - name: i3c-device.yaml
```

Comment:
remove this commit...
you are still using the wrong compat, `vnd,i3c-device` has what you should be using

Reply:
I included this file because the "supports-setaasa" property is required for my i3c samples.

Reply:
When I used the compat "vnd,i3c-device" under samples, I encountered a compilation error. The reason for this error was that we do not have the creation of the device object from the device tree node identifier for this compat.

Using compat as  "vnd,i3c" worked with me under samples because we already have the creation of the device object for this compatibility under the drivers/i3c_test file.

Reply:
Remove this commit again, Please see my earlier comments... you are still doing the same issue I mentioned earlier